config/examples/redhat_customer_portal.yml

```
@@ -1,19 +1,15 @@
 base: &default
-  enabled: true
-  client_id:
-  client_secret:
-  realm:
-  flow:
-  entitlements_notifications_enabled: true
+  enabled: <%= ENV.fetch('RH_CUSTOMER_PORTAL_ENABLED', false) %>
+  client_id: <%= ENV.fetch('RH_CUSTOMER_PORTAL_CLIENT_ID', '3scale') %>
+  client_secret: <%= ENV['RH_CUSTOMER_PORTAL_CLIENT_SECRET'] %>
+  realm: <%= ENV.fetch('RH_CUSTOMER_PORTAL_REALM', 'implicit') %>
+  flow: <%= ENV.fetch('RH_CUSTOMER_PORTAL_FLOW', 'implicit') %>
+  entitlements_notifications_enabled: <%= ENV.fetch('RH_SUPPORT_ENTITLEMENTS_NOTIFICATIONS', '1') == '1' %>
```

```suggestion
  entitlements_notifications_enabled: <%= ActiveModel::Type::Boolean.new.cast(ENV.fetch('RH_SUPPORT_ENTITLEMENTS_NOTIFICATIONS', '1')) %>
```

We use `== '1'` at many places. But we also use `ActiveModel::Type::Boolean.new.cast` at two. I wonder if we should start using it more or keep `== '1'`. So this is an open question.

---

Actually, after thinking about it again - indeed, with `== '1'` we only expect `1` as the truthy value, `true` will not work.
I guess it depends on whether we want to support `true` as a truthy value or not.

---

that's exactly why I said it was an open question :)